src/themes/ec/_variables.scss

```diff
@@ -642,6 +642,16 @@ $range: (
 );
 /* stylelint-enable scss/no-global-function-names */
 
+// Rating field
+$rating-field: (
+  checked-color: var(--ecl-color-secondary-100),
+  stroke-color: var(--ecl-color-dark-100),
+  hover-stroke-color: var(--ecl-color-primary-80),
+  checked-hover-color: var(--ecl-color-secondary-140),
+  checked-focus-stroke: var(--ecl-color-dark-100),
+  focus-circle: true,
```

Comment:
I guess that `focus-circle` is no longer used

Reply:
it is used, not in the way it was, but still to distinguish the two systems and apply the outline on focus in EU. I agree with you that removing any style on focus might be questionable, but since they rejected their own request of a focus style for "design" reasons, i guess that applying a style to the whole element would make it even worst.